nifi-assembly/NOTICE

```diff
@@ -193,6 +193,8 @@ The following binary components are provided under the Apache Software License v
 
   (ASLv2) opencsv (net.sf.opencsv:opencsv:2.3)
 
+  (ASLv2) Super CSV (net.sf.supercsv:super-csv:2.4.0)
```

Comment:
This is a Apache 2.0 licensed import with not NOTICE, it can be used with adding anything to the NOTICE or LICENSE file. This applies to this assembly NOTICE and the nar NOTICE.